packages/headless/src/test/mock-commerce-search.ts

```diff
@@ -1,3 +1,18 @@
+import {SearchCommerceSuccessResponse} from '../api/commerce/search/response';
+import {QuerySearchCommerceAPIThunkReturn} from '../features/commerce/search/search-actions';
 import {buildFetchProductListingV2Response} from './mock-product-listing-v2';
 
-export {buildFetchProductListingV2Response as buildSearchResponse};
+export function buildSearchResponse(
+  response: Partial<SearchCommerceSuccessResponse> = {},
+  originalQuery: string = '',
+  queryExecuted: string = ''
+): QuerySearchCommerceAPIThunkReturn {
+  return {
+    response: {
+      ...buildFetchProductListingV2Response(response).response,
```

Comment:
We should rename that function, it's silly 😅 

Maybe `buildFetchProductsCoreResponse` ? 

Reply:
I've aliased it to `buildFetchProductsCoreResponse` in the import of this file since we're only using this function as core for the search response in this specific file. In other parts, we're using it as a way to build the listing response.